h/script.py

```diff
@@ -118,12 +133,18 @@ def token(args):
 
     print(token)
 
-parser_token = subparsers.add_parser('token', help=token.__doc__)
+parser_token = subparsers.add_parser(
+    'token', description=token.__doc__,
```

Comment:
If you could employ `textwrap.dedent` here ([c.f. `hypothesis-buildext`](https://github.com/hypothesis/h/blob/9f9496198e8177653dba5781fcbefed57f878be8/h/buildext.py#L267)) and reformat the docstring to look a little more conventional, that would make me happy :).